manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-integration-tests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ocp4-stage-thoth-integration-tests
+spec:
+  project: thoth-stage
+  source:
+    repoURL: 'https://github.com/thoth-station/thoth-application.git'
+    path: integration-tests/overlays/ocp4-stage
+    targetRevision: HEAD
```

Comment:
i noticed this is `HEAD` while test is pointed at `master`.  Is this intentional? (i think this means you would be changing HEAD remote refs to alter the target commit)